datahub-web-react/src/app/entity/shared/EntityDropdown/CreateGlossaryEntityModal.tsx

```diff
@@ -32,21 +39,27 @@ function CreateGlossaryEntityModal(props: Props) {
     const entityRegistry = useEntityRegistry();
     const [stagedName, setStagedName] = useState('');
     const [selectedParentUrn, setSelectedParentUrn] = useState(entityData.urn);
+    const [documentation, setDocumentation] = useState('');
+    const [isDocumentationModalVisible, setIsDocumentationModalVisible] = useState(false);
     const [createButtonDisabled, setCreateButtonDisabled] = useState(true);
     const refetch = useRefetch();
 
     const [createGlossaryTermMutation] = useCreateGlossaryTermMutation();
     const [createGlossaryNodeMutation] = useCreateGlossaryNodeMutation();
 
+    const sanitizedDocumentation = DOMPurify.sanitize(documentation);
```

Comment:
this is cool! what is this DOMPurify thing doing?

Reply:
it's protecting against something like a XSS attack by preventing us from sending back and rendering unsafe html! DOMPurify has a built in list of do-not-allow tags and what not so that users can't try to submit a `<script>{do_something}</script>` that could then get rendered and be no bueno. So we sanitize markdown before sending to the backend, but just as importantly, before rendering the markdown on the page.

Reply:
i'm realizing now that I don't really need this sanitization here though, as I'm no longer rendering this rich text in the modal like I was before (when I demo'd) so I can remove! however I still want it before sending to backend. and we use this bad boy elsewhere when we actually render rich text